4-Classification/2-Classifiers-1/translations/README.es.md

```diff
@@ -0,0 +1,245 @@
+
+
+Scikit-learn ofrece esta tabla para explicar como los solucionadores manejan distintos desafíos presentados por distintas clases de datos estructurados:
+
+![solucionadores](../images/solvers.png)
+
+## Ejercicio - divide los datos
+
+Nos podemos enfocar en la regresión logística para nuestra primer prueba de entrenamiento ya que recién aprendiste sobre esto último en la lección anterior.
+Divide tus datos en los grupos 'training' y 'testing' al llamar a `train_test_split()`:
+
+```python
+X_train, X_test, y_train, y_test = train_test_split(cuisines_feature_df, cuisines_label_df, test_size=0.3)
+```
+
+## Ejercicio - aplica la regresión logística
+
+Ya que estás usando un caso multiclase, necesitas elegir qué _esquema_ usar y qué _solucionador_ configurar. Usa LogisticRegression con un ajuste multiclase y el solucionador **liblinear** para entrenar.
+
+1. Crea un regresión logística con un multi_class configurado a `ovr` y el solucionador ajustado a `liblinear`:
+
+    ```python
+    lr = LogisticRegression(multi_class='ovr',solver='liblinear')
+    model = lr.fit(X_train, np.ravel(y_train))
+    
+    accuracy = model.score(X_test, y_test)
+    print ("Accuracy is {}".format(accuracy))
+    ```
+
+    ✅ Prueba un solucionador diferente como `lbfgs`, el cual suele ser configurado por defecto
+
+    > Nota, usa la función de Pandas [`ravel`](https://pandas.pydata.org/pandas-docs/stable/reference/api/pandas.Series.ravel.html) para aplanar tus datos cuando sea necesario.
+
+    ¡La precisión es buena por enciam del **80%*!
+
+1. Puedes ver este modelo en acción al probar una fila de datos (#50):
+
+    ```python
+    print(f'ingredients: {X_test.iloc[50][X_test.iloc[50]!=0].keys()}')
+    print(f'cuisine: {y_test.iloc[50]}')
+    ```
+
+    El resultado es impreso:
+
+   ```output
+   ingredients: Index(['cilantro', 'onion', 'pea', 'potato', 'tomato', 'vegetable_oil'], dtype='object')
+   cuisine: indian
+   ```
+
+   ✅ Prueba un número de fila distinto y revisa los resultados
+
+1. Indagando más, puedes revisar la precisión de esta predicción:
+
+    ```python
+    test= X_test.iloc[50].values.reshape(-1, 1).T
+    proba = model.predict_proba(test)
+    classes = model.classes_
+    resultdf = pd.DataFrame(data=proba, columns=classes)
+    
+    topPrediction = resultdf.T.sort_values(by=[0], ascending = [False])
+    topPrediction.head()
+    ```
+
+    El resultado es impreso - La cocina India es su mejor conjetura, con buena probabilidad:
+
+    |          |        0 |
+    | -------: | -------: |
+    |   indian | 0.715851 |
+    |  chinese | 0.229475 |
+    | japanese | 0.029763 |
+    |   korean | 0.017277 |
+    |     thai | 0.007634 |
+
+    ✅ ¿Puedes explicar por qué el modelo está muy seguro esta es una cocina India?
```

Comment:
To sound more natural, it can say "¿Puedes explicar por qué el modelo está muy seguro de que esta es una cocina India?"